.gitignore

```diff
@@ -6,6 +6,10 @@
 *.dylib
 kubevirt-csi-driver
 
+#k8s e2e suite
+.kvcluster-kubeconfig-e2e
+capk.pem
```

Comment:
do we still need this?

Reply:
technically I do remove the capk.pem that I push into a secret during the script run. But if something goes wrong and it doesn't get deleted, it makes sense to not accidentally add it to the repo.